.github/PULL_REQUEST_TEMPLATE/pull_request_template_bug-fix.md

```diff
@@ -0,0 +1,48 @@
+<!-- Please, fill in the description as completely as possible.-->
+
+### Description
+
+<!-- Explain the issue with the bug (what part of the code, what are the side effects of the bug)
+       How did the bug was found, do you know what commit introduced the bug? -->
+
+### Solution
+
+<!-- How did you fix the bug?
+       Is it a permanent or temporary fix? (if temporary, please open an issue) -->
+
+### Testing
+
+<!-- How has this been tested?
+       What are the new test(s) that reproduce the bug?
+       Are there changes and/or impacts on current tests, why?
+       How did you ensure that the solution works? -->
+
+### Documentation
+
+<!-- Does this fix modify or have new simulation parameters? If so, describe them. -->
+
+### Miscellaneous (will be removed when merged)
+
+<!-- Anything that you would like to add that does not fit into another category
+       Examples:
+         Future changes or features that will be added in subsequent pull requests
+         Any comments or highlights for the reviewers -->
+
+### Checklist (will be removed when merged)
+See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.
+*If applicable
+
+Code related list:
+- [ ] All in-code documentation related to this PR is updated (Doxygen format)*
+- [ ] Lethe documentation is updated*
+- [ ] Fix has unit test(s) (preferred) or application test(s)
+- [ ] The branch has been rebased onto master
+- [ ] Changelog (CHANGELOG.md) is updated
+- [ ] Code is indented with indent-all and .prm files (examples and tests) with prm-indent
```

Comment:
(same for other files)

```suggestion
- [ ] All in-code documentation related to this PR has been updated (Doxygen format)*
- [ ] Lethe documentation has been updated*
- [ ] Unit test(s) (preferred) or application test(s) have been added to test the fix
- [ ] The branch has been rebased onto master
- [ ] Changelog (CHANGELOG.md) has been updated
- [ ] Code has been indented with indent-all and .prm files (examples and tests) with prm-indent
```

Reply:
I must say that I don't fully agree with those changes, but let's discuss here

For me, giving the statement that something have been done means that an action related to modification has been perform. But, what I wanted the checklist to focus on, is the state of the thing and not the process.

For instance with the documentation:
Lethe documentation is updated vs. Lethe documentation has been updated
For me, #1 means the author made sure that the documentation is updated related to its work, maybe the person had to modify documentation or not. 
#2 means that the author had to modify the documentation.

Maybe it is a language interpertation thing, but I think "is" is better
However, it highlights the irrevelancy of the *if application, that I will remove.

What do you think about it :) ?